Src/Newtonsoft.Json/NameTable.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Newtonsoft.Json
+{
+    /// <summary>
+    /// The abstract name table.
```

Comment:
Base class for a table of atomized string objects.

Reply:
Done